nf_core/create.py

```diff
@@ -35,21 +40,164 @@ class PipelineCreate(object):
         outdir (str): Path to the local output directory.
     """
 
-    def __init__(self, name, description, author, version="1.0dev", no_git=False, force=False, outdir=None):
-        self.short_name = name.lower().replace(r"/\s+/", "-").replace("nf-core/", "").replace("/", "-")
-        self.name = f"nf-core/{self.short_name}"
-        self.name_noslash = self.name.replace("/", "-")
-        self.name_docker = self.name.replace("nf-core", "nfcore")
-        self.logo_light = f"{self.name_noslash}_logo_light.png"
-        self.logo_dark = f"{self.name_noslash}_logo_dark.png"
-        self.description = description
-        self.author = author
-        self.version = version
+    def __init__(
+        self,
+        name,
+        description,
+        author,
+        version="1.0dev",
+        no_git=False,
+        force=False,
+        outdir=None,
+        template_yaml_path=None,
+        plain=False,
```

Comment:
You've got a point, but on the other hand, prompting by default showcases the functionality better. I think that since the default option is to not customize the template, it is easy enough to create a nf-core pipeline.

Reply:
Doesn't really matter, but semantically I would usually use `None` for "no default" and prompt only if set to that.

Only really matters if people use the function in custom Python scripts.